lib/versioning/VersionID.js

```diff
@@ -9,12 +9,12 @@
 // the lengths of the components in bytes
 const LENGTH_TS = 14; // timestamp: epoch in ms
 const LENGTH_SEQ = 6; // position in ms slot
-const LENGTH_SITE = 5; // site identifier
+const LENGTH_TOKEN = 5; // replication group identifier
 
 // empty string template for the variables in a versionId
 const TEMPLATE_TS = new Array(LENGTH_TS + 1).join('0');
 const TEMPLATE_SEQ = new Array(LENGTH_SEQ + 1).join('0');
-const TEMPLATE_SITE = new Array(LENGTH_SITE + 1).join(' ');
+const TEMPLATE_TOKEN = new Array(LENGTH_TOKEN + 1).join(' ');
```

Comment:
I don't see why you need to change the name of the internal variable; using the name you prefer in Federation is enough.